### PR TITLE
[sysdig/kspm-collector/node-analyzer/sysdig-deploy] Removing duplicate labels

### PR DESCRIPTION
**Pull request by mikescholl-sysdig**
## What this PR does / why we need it:

Removes duplicate labels from the following charts
- kspm-collector
- agent
- node-analyzer

## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Title of the PR starts with chart name (e.g. [mychartname])
- [x] Chart Version bumped
- [x] Variables are documented in the README.md (or README.tpl in some charts)
- [x]  Check GithubAction checks (like lint) to avoid merge-check stoppers

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/sysdig/CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,15 @@
 
 This file documents all notable changes to Sysdig Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
-## v1.15.19
+## v1.15.20
+### Bugfixes
+* Removed duplicate labels from deployment of `app.kubernetes.io/instance` 
+
+## v1.15.20
```
Comment (lilx1ao) [inline charts/sysdig/CHANGELOG.md]: this should be 19

File: charts/sysdig-deploy/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,13 @@
 
 This file documents all notable changes to Sysdig's sysdig-deploy Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.1.5
+
+### Bugfixes
+
+* Bumped node-analyzer to 1.5.9
+* Bumped kspm-collector to 0.5.9
```
Comment (lilx1ao) [inline charts/sysdig-deploy/CHANGELOG.md]: not quite

**Comment by lilx1ao** [comment]
I like how many chart names you can fit into that bracket in the pr title.

**Comment by mikescholl-sysdig** [comment]
@lilx1ao gotta fit em all 😂 

Updated the versions if you want to review again. TY!

File: charts/sysdig-deploy/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,13 @@
 
 This file documents all notable changes to Sysdig's sysdig-deploy Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.1.5
```
Comment (yashgiri) [inline charts/sysdig-deploy/CHANGELOG.md]: v1.1.7*

File: charts/sysdig-deploy/CHANGELOG.md (hunk 2 above)
Comment (lilx1ao) [reply charts/sysdig-deploy/CHANGELOG.md]: no but this still isn't correct

File: charts/sysdig-deploy/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,13 @@
 
 This file documents all notable changes to Sysdig's sysdig-deploy Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.1.7
+
+### Bugfixes
+
+* Bumped node-analyzer to 1.5.10
+* Bumped kspm-collector to 0.5.9
```
Comment (achandras) [inline charts/sysdig-deploy/CHANGELOG.md]: Shouldn't this be `0.1.5` as per the [Chart.yaml bump](https://github.com/sysdiglabs/charts/pull/539/files#diff-f655c9de8ab8071079b19bc9d8717dbc40206918a3acd44ecc57349e5c2da5ddR5) above?

File: charts/sysdig-deploy/CHANGELOG.md (hunk 2 above)
Comment (achandras) [reply charts/sysdig-deploy/CHANGELOG.md]: Oops I see you've already mentioned this @lilx1ao

Comment (lilx1ao) [reply charts/sysdig-deploy/CHANGELOG.md]: yeah. lets hope Mike sees it the THIRD time lol

File: charts/sysdig-deploy/CHANGELOG.md (hunk 4 above)
Comment (mikescholl-sysdig) [reply charts/sysdig-deploy/CHANGELOG.md]: Yes; updated @achandras

**Review by lilx1ao — APPROVED** [review]

**Pull request merged**